hummingbot/connector/derivative/binance_perpetual/binance_perpetual_derivative.py

```diff
@@ -134,9 +135,9 @@ def __init__(self,
         self._user_stream_event_listener_task = None
         self._trading_rules_polling_task = None
         self._funding_info_polling_task = None
+        self._funding_fee_polling_task = None
         self._last_poll_timestamp = 0
         self._throttler = Throttler((10.0, 1.0))
-        self._funding_payment_span = [0, 15]
```

Comment:
you shouldn't remove this. Some strategies will have issues.

Reply:
Fixed in latest commit